src/protocol.h

```diff
@@ -263,7 +263,7 @@ enum ServiceFlags : uint64_t {
     NODE_BLOOM = (1 << 2),
     // NODE_WITNESS indicates that a node can be asked for blocks and transactions including
     // witness data.
-    NODE_WITNESS = (1 << 3),
```

Comment:
If this is intended to replace the existing segwit sf then why would it be confusing?